mgo.go

```diff
@@ -306,7 +306,9 @@ func (inst *MgoInstance) run() error {
 			listening <- nil
 		} else {
 			err := fmt.Errorf("mongod failed to listen on port %v", mgoport)
-			if strings.Contains(buf.String(), "addr already in use") {
+			out := buf.String()
+			if strings.Contains(out, "addr already in use") ||
+				strings.Contains(out, "Address already in use") {
```

Comment:
What's the chances, `strings.Contains(out, "already in use")` would be a better fit?

Reply:
Yeah good point, I thought I should be specific in-case behaviour changed again in Mongo, but it's only for testing.